ctapipe_io_nectarcam/__init__.py

```diff
@@ -80,59 +131,61 @@ def __init__(self, **kwargs):
         self.data = None
         self.log.info("Read {} input files".format(self.multi_file.num_inputs()))
         self._tel_id = self.camera_config.telescope_id
-        self._subarray_info = self.prepare_subarray_info(self._tel_id)
+        self.geometry_version = 3
+        self._subarray = self.create_subarray(self.geometry_version, self._tel_id)
 
     @property
-    def subarray(self):
-        return self._subarray_info
-
+    def is_simulation(self):
+        return False
 
+    @property
+    def datalevels(self):
+        return (DataLevel.R0, DataLevel.R1)
```

Comment:
Does it really contain R1? So do you perform all the needed low-level corrections and transform from adc counts to pe?

Reply:
No, to my understanding only a (fixed) pedestal subtraction is performed. The purpose of this PR is to have a working version of the io plugin for ctapipe 0.12, so that further changes/additions needed can be implemented later.

Reply:
But then the correct thing to do would only add the `R0` datelevel

Reply:
R1 requires both pedestal subtraction and gain correction (PE/DC at least, but also a rough relative gain factor if needed). See the R1-Event data model document (CTA-SPE-COM-000000-0002 revision 1f or later)

Reply:
Yes, I understand that, so the correct way to implement that for the moment would be

```
@property
    def datalevels(self):
        return (DataLevel.R0)
```

Is that right?

Reply:
@tibaldo Careful with tuples that only have one element, you need a trailing comma:

```
return (DataLevel.R0, )
```